app/models/customer.rb

```diff
@@ -1,2 +1,5 @@
 class Customer < ApplicationRecord
+  validates :first_name, :last_name, presence: true
+  has_many :invoices
+  # has_many :transactions, through: :invoices
```

Comment:
Wasn't totally sure about this relationship, couldn't get the test to pass but wanted to leave it just in case it comes into play later.